src/app/learn/get-started/advanced/accessibility-unit-tests/index.html

```diff
@@ -32,70 +32,41 @@
+    After you import the testing library, the next step to run accessibility checks in unit tests is to import the extended Jasmine <sky-code>expectAsync</sky-code> function into <sky-code>.spec.ts</sky-code> files. At the top of the spec files for the components to test, import the function from the testing library:
   </p>
   <sky-code-block>
     import {
-      expect
+      expectAsync
     } from '@skyux-sdk/testing';
   </sky-code-block>
   <p>
-    After you import <sky-code>expect</sky-code> function, you can use it to run accessibility tests for components. For components that change state, you can insert the function into existing <sky-code>it</sky-code> blocks. And for components that don't change state, the function can stand alone to run accessibility checks from its own <sky-code>it</sky-code> block.
+    After you import <sky-code>expectAsync</sky-code> function, you can use it to run accessibility tests for components. For components that change state, you can insert the function into existing <sky-code>it</sky-code> blocks. And for components that don't change state, the function can stand alone to run accessibility checks from its own <sky-code>it</sky-code> block.
   </p>
 
   <stache-page-anchor>
-    Insert the <sky-code>expect</sky-code> function and <sky-code>toBeAccessible</sky-code> matcher
+    Insert the <sky-code>expectAsync</sky-code> function and <sky-code>toBeAccessible</sky-code> matcher
   </stache-page-anchor>
 
   <p>
-    After you import the <sky-code>expect</sky-code> function into spec files, the next step is to wrap the <sky-code>expect</sky-code> function around the elements to test for accessibility. Accessibility checks are asynchronous, so it's important to wrap your tests in Angular's <sky-code>async</sky-code> method. And to break the linear flow of control when using <sky-code>async</sky-code>, it's also important to wrap <sky-code>toBeAccessible</sky-code> in Angular's <sky-code>whenStable</sky-code> method.
+    After you import the <sky-code>expectAsync</sky-code> function into spec files, the next step is to wrap the <sky-code>expectAsync</sky-code> function around the elements to test for accessibility.
   </p>
 
   <sky-code-block>
-    it('should pass accessibility', async(() => {
-      fixture.whenStable().then(() => {
-        expect(element).toBeAccessible();
-      });
+    it('should pass accessibility', async () => {
+      await expectAsync(element).toBeAccessible();
     }));
   </sky-code-block>
 
-  <p>
-    Keep in mind that <sky-code>async</sky-code> is also a native JavaScript declaration, so you need to explicitly import the Angular <sky-code>async</sky-code> method at the top of the unit test spec file.
-  </p>
-
-  <sky-code-block>
-    import { async } from ‘@angular/core/testing’;
-  </sky-code-block>
-
   <stache-page-anchor>
     Specify parameters for <sky-code>toBeAccessible</sky-code>
```

Comment:
```suggestion
    Overwrite accessibility checks
```

Note: I'd also delete the current subheader that says `Overwrite accessibility checks`. If not, I wouldn't make this change. But it doesn't seem like we need the subheader anymore, and it seems like this header is inaccurate now that we aren't talking about plural `parameters`.